packages/website/src/shared/lib/utilities/testing/testing.tsx

```diff
@@ -1,12 +1,15 @@
-import { render as renderRTL } from "@testing-library/react";
 import { PropsWithChildren, ReactElement } from "react";
+import { render as renderRTL } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
 import TestWrapper from "./test-wrapper";
 
 const createWrapper =
   (props = {}) =>
   ({ children }: PropsWithChildren) => <TestWrapper {...props}>{children}</TestWrapper>;
 
-export const render = (ui: ReactElement, { routes = ["/"], ...options } = {}) =>
-  renderRTL(ui, { wrapper: createWrapper({ routes }), ...options });
+export const render = (ui: ReactElement, { routes = ["/"], ...options } = {}) => ({
```

Comment:
I don't understand this? What is the purpose of creating render wrapper?

Reply:
The wrapper surrounds the component that is being tested with dependencies, like the router. In the future, we would include the store, as well. It sort of like dependency injection. Otherwise, we would need to manually create a router (and a store, etc.) every time.